std/uni.d

```diff
@@ -2318,7 +2318,7 @@ public:
      * $(LI $(B %x) formats the intervals as a [low..high$(RPAREN) range of lowercase hex characters)
      * $(LI $(B %X) formats the intervals as a [low..high$(RPAREN) range of uppercase hex characters)
      */
-    void toString(scope void delegate(const(char)[]) sink,
+    void toString(Writer)(scope Writer sink,
```

Comment:
I'm not sure it matters. template constraints are meant to allow you to provide other template overloads based on some constraint. But this is the only `toString` member.